### DISCUSSION TITLE
Nextcloud Office app always gets uninstalled - Using Collabora App with external Collabora Code Server

### DISCUSSION THREAD

**Discussion by MasterxAndi** [nextcloud/all-in-one#3898] [Questions] [answered]
Hi,

when I click "Download and activate" Nextcloud Office, I can configure Collabora and it works.

After the Backup (Recreation of contianers maybe?!) 
It disables and delets the app.
I have to Download and Activate in the appstore manually again.

How can I keep the Office App up and running?

**Comment by szaimen** [comment, marked as answer]
> How can I keep the Office App up and running?

Enable collabora in thr AIO interface.

Apart from that see https://github.com/nextcloud/all-in-one#how-to-keep-disabled-apps

**Reply by AndiDoe** [reply]
Yes, but I don't want an extra Container on this Server running....

At the moment I added the reinstall to cron on this Server

Other posibilities?

**Reply by szaimen** [reply]
See the link I sent...

**Reply by AndiDoe** [reply]
Many Thanks.
;)
Thats what I searched for...
